.github/workflows/build.yml

```diff
@@ -113,28 +99,27 @@ jobs:
     steps:
       - uses: cisagov/setup-env-github-action@develop
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
         with:
           python-version: 3.8
       - name: Store installed Python version
         run: |
           echo "::set-env name=PY_VERSION::"\
           "$(python -c "import platform;print(platform.python_version())")"
-      - name: Cache pip test requirements
-        uses: actions/cache@v1
+      - name: Cache testing environments
+        uses: actions/cache@v2
         with:
-          path: ${{ env.PIP_CACHE_DIR }}
-          key: "${{ runner.os }}-pip-test-py${{ env.PY_VERSION }}-\
+          path: |
+            ${{ env.PIP_CACHE_DIR }}
+            ${{ env.CURL_CACHE_DIR }}
+          key: "test-${{ runner.os }}-\
+            py${{ env.PY_VERSION }}-\
+            packer${{ env.PACKER_VERSION }}-\
```

Comment:
`packer` is written in `go` just like `terraform`.  Why do we cache `go` above but not here?

Reply:
Nevermind.  I realize now that `go` is cached above because `go` modules are used by some of our `pre-commit` hooks.